### PR TITLE
fix to use mysql

### PR DESCRIPTION
**Pull request by jedmccaleb**

### CODE REVIEW

File: infra/jobczar/requesthandler.go [hunk 1]
```diff
@@ -30,20 +36,25 @@ func (handler RequestHandler) FetchWork(ctx context.Context, workerName string,
...
+func (handler RequestHandler) SubmitResult_(ctx context.Context, result *infra.ResultWork) (bool, error) {
 	println("SUBMIT RESULT")
 
-	sql := "UPDATE jobs set cycles=$1,time_start=$2,timeStop=$3,score=$4 where job_id=$5"
-	_, err := gDatabase.db.Exec(sql, result.Cycles, result.TimeStart, result.TimeStop, result.Score, result.JobID)
-	if err != nil {
-		fmt.Println(err)
-		return false, err
+	if result.Status == goodJob {
+		sql := fmt.Sprintf("UPDATE jobs set cycles=%d,time_start=%d,time_end=%d,score=%f where job_id=%d", result.Cycles, result.TimeStart, result.TimeStop, result.Score, result.JobID)
```
Comment (etuleu) [inline infra/jobczar/requesthandler.go:49]: are we missing status here?

File: infra/jobczar/requesthandler.go [hunk 2]
```diff
@@ -53,8 +64,8 @@ func (handler RequestHandler) AddJob(ctx context.Context, agentName string, worl
 	agentCfg string, worldCfg string, priority int32, userID int32) (int32, error) {
 	println("job czar")
 
-	sql := "INSERT into jobs (user_id,priority,agent_name,world_name,agent_param,world_param) values ($1,$2,$3,$4,$5,$6)"
-	_, err := gDatabase.db.Exec(sql, userID, priority, agentName, worldName, agentCfg, worldCfg)
+	sql := fmt.Sprintf("INSERT into jobs (user_id,priority,agent_name,world_name,agent_param,world_param) values (%d,%d,`%s`,`%s`,`%s`,`%s`)", userID, priority, agentName, worldName, agentCfg, worldCfg)
```
Comment (etuleu) [inline infra/jobczar/requesthandler.go:67]: we should probably be using some kind of helper method that escapes sql statements?

File: infra/jobczar/requesthandler.go (hunk 1 above)
Comment (jedmccaleb) [reply infra/jobczar/requesthandler.go]: thanks. fixed

File: infra/jobczar/requesthandler.go (hunk 2 above)
Comment (jedmccaleb) [reply infra/jobczar/requesthandler.go:67]: maybe. you can add an issue for this. I don't think we should hold up the PR for it though

**Pull request merged**